### ISSUE TITLE
Extracting variable info from a function when using bot.gateway.command(), as if you were using return

### ISSUE THREAD

**Issue by Glennvb85** [Merubokkusu/Discord-S.C.U.M#423]
I want send a selfbot slash command and use a function every time a response is given after that initial slash command is sent. 

Then listen to every response sent by using `bot.gateway.command` , and use my own function to extract data. how can I save the information I placed after the `return` statement, so that the next time `bot.gateway.command` triggers my function again, the result is updated. 

```
import discum

userToken = '123124123512'

#Initial parameters
CommandId = ''
Progression = ''
url = ''
timeStamp =''
requestContent = ''
botID = ''

bot = discum.Client(token=userToken)

def myFunction(resp,CommandId,Progression,url,timeStamp,requestContent):
    if resp.event.ready_supplemental:       #
        user = bot.gateway.session.user
        print("Logged in as {}#{}".format(user['username'], user['discriminator']))
        print("running slash command")
        Runcommand(inputTextCommand="hello world")  #This custom command runs a selfbot slash command
        return ("I WANT TO SAVE THIS INFORMATION")

    #parsing response result
    parsedResp = resp.parsed.auto()
    
    #check if response is from right bot id and user is mentioneded
    if parsedResp['webhook_id'] == botID and  parsedResp['mentions'][0]['username'] == bot.gateway.session.user['username']:
        print("Slash command response targeted to session user")
        CommandId = parsedResp['interaction']['id']
        url = parsedResp['attachments'][0]['url']
        return CommandId, url           #HOW DO I the returned information so I can use it for future responses
    else:

 

bot.gateway.command(
    {
        "function": myFunction,
        "Priority": 0,
        "params": {"CommandId": CommandId, "Progression": Progression, "url":url, "timeStamp":timeStamp, "requestContent":requestContent}
    }
)




bot.gateway.run()
```

In other words I want to achieve something like this:

```
state = False

def foo(resp,state):
    if resp.event.ready_supplemental:
       state== True
       return state

    if state == True:
        print("My state is True! ")
        return


state = bot.gateway.command(
    {
        "function": Foo,
        "Priority": 0,
        "params": {"state": state, 
    }
)
    


```

**Comment by NeutronBlast** [comment]
It's possible to extract info from something that happens during a gateway function but not with return. I needed to do something similar and I'd just set variables inside the bot object because they'd be updated. So instead of 

`return CommandId, url`

I would have something like
```
bot.__setattr__('command_id', CommandId)
bot.__setattr__('url', url)
```

Then, if you want to get this information later you'd just do

`
bot.__getattribute__('command_id')
`

Hope it helps

**Comment by Glennvb85** [comment]
I will give that a go. thank  you!
